Dockerfile

```diff
@@ -1,34 +1,23 @@
-FROM centos:7
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
 
-RUN yum -y update && \
-    yum -y install java-1.8.0-openjdk
+RUN useradd -M tomcat
 
-ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.292.b10-1.el7_9.x86_64
-
-ENV JRE_HOME /usr/lib/jvm/jre
-
-RUN useradd -r -g root tomcat
-
-
-ADD  http://apache.cs.utah.edu/tomcat/tomcat-9/v9.0.46/bin/apache-tomcat-9.0.46.tar.gz /tmp
+ENV TOMCAT_VERSION=9.0.22  \
+    TOMCAT_MAJOR_VERSION=9
+    
+ADD  https://archive.apache.org/dist/tomcat/tomcat-${TOMCAT_MAJOR_VERSION}/v${TOMCAT_VERSION}/bin/apache-tomcat-${TOMCAT_VERSION}.tar.gz /tmp
 
 RUN cd /tmp && \
-    tar -xf apache-tomcat-9.0.46.tar.gz && \
-    mv apache-tomcat-9.0.46 /opt/tomcat/ && \
-    chown -R tomcat: /opt/tomcat && \
-    sh -c 'chmod +x /opt/tomcat/bin/*.sh'
-
-ENV CATALINA_HOME /opt/tomcat/
-ENV CATALINA_PID /opt/tomcat/temp/tomcat.pid
+    tar -xf apache-tomcat-${TOMCAT_VERSION}.tar.gz && \
+    rm apache-tomcat-${TOMCAT_VERSION}.tar.gz && \
+    mv apache-tomcat-${TOMCAT_VERSION} /opt/tomcat/ && \
+    chown -R tomcat:tomcat /opt/tomcat && \
```

Comment:
Please `chown -R root:root /opt/tomcat` and give write permissions only where absolutely needed by Tomcat.